packages/reputation-miner/bin/index.js

```diff
@@ -35,25 +35,49 @@ const {
   adapterLabel,
 } = argv;
 
+class RetryProvider extends ethers.providers.JsonRpcProvider {
+  constructor(url, adapterObject){
+    super(url);
+    this.adapter = adapterObject;
+  }
 
-if ((!minerAddress && !privateKey) || !colonyNetworkAddress || !syncFrom) {
-  console.log("❗️ You have to specify all of ( --minerAddress or --privateKey ) and --colonyNetworkAddress and --syncFrom on the command line!");
-  process.exit();
-}
+  static attemptCheck(err, attemptNumber){
+    if (attemptNumber === 11){
```

Comment:
Why this value?

Reply:
The backoff module tries ten times. If it's going to fail out entirely, my understanding is that this function will be called with `attemptNumber` equal to eleven and I want to see what went so wrong in the logs. If it's retrying but still might succeed, no need to log.

Reply:
Can we indicate that with some constant somewhere? E.g. `BACKOFF_ATTEMPTS + 1`